tensorflow/tools/ci_build/windows/cpu/pip/build_tf_windows.sh

```diff
@@ -46,10 +46,9 @@ clean_output_base
 
 run_configure_for_cpu_build
 
-BUILD_OPTS='-c opt --cpu=x64_windows_msvc --host_cpu=x64_windows_msvc --copt=/w --verbose_failures --experimental_ui'
-
 bazel build $BUILD_OPTS tensorflow/tools/pip_package:build_pip_package || exit $?
 
+rm -f ./tensorflow-*.whl
```

Comment:
same comment as in GPU script.
let's build the pip package into a new directory. and avoid rm -f here.